.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment (please complete the following information):**
+ - Device: [e.g. Desktop, iPhone 8]
+ - OS: [e.g. Windows 10, iOS 11]
+ - Browser: [e.g. Chrome, Safari]
+ - Version: [e.g. 72]
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
I guess people can add a link here if they choose to.

Reply:
I'll add:

> If your installation is available on the web or you can provide a minimal repo that reproduces the bug, please add the link to it.

Reply:
Sounds good.  Thanks for adding that.